pandas/_libs/tslibs/resolution.pyx

```diff
@@ -373,21 +373,27 @@ cdef unique_deltas(ndarray[int64_t] arr):
     return result
 
 
-def _is_multiple(us, mult):
+cdef inline bint _is_multiple(int64_t us, int64_t mult):
     return us % mult == 0
 
 
-def _maybe_add_count(base, count):
+cdef inline str _maybe_add_count(str base, count):
     if count != 1:
         return '{count}{base}'.format(count=int(count), base=base)
```

Comment:
If above comment is correct shouldn't need the int cast here

Reply:
Looks like you're right.  Just pushed a commit that added the suggested typing and removed the `int(`.  (and hopefully fixed the test errors)